lib/rules/no-unsupported-features/es-syntax-additions.json

```diff
@@ -0,0 +1,30 @@
+{
```

Comment:
This is currently entirely unused. I wanted feedback on the other changes first!